### ISSUE TITLE
TypeDoc doesn't work anymore after I upgraded my package.json to TypeScript 3.7.2

### ISSUE THREAD

**Issue by fr0** [TypeStrong/typedoc#1127]
- [x] I have checked [issues with bug label](https://github.com/TypeStrong/typedoc/labels/bug) and found no duplicates

## Expected Behavior

It should let me use a newer version of TypeScript as soon as it is released.

## Actual Behavior

It throws an error:

```
Using TypeScript 3.5.3 from /home/jenkins/workspace/Atlas_atlas-dev-api_master/node_modules/typedoc/node_modules/typescript/lib
Error: /home/jenkins/workspace/Atlas_atlas-dev-api_master/node_modules/atlas-client-web/atlas-application.d.ts(93)
 An accessor cannot be declared in an ambient context.
```

## Steps to reproduce the bug

I'm not yet using any new features from TypeScript 3.7. The line of code it is complaining about is a `get` accessor in one of my project's dependencies (which was also compiled using TS 3.7)

## Environment
  - Typedoc version:
  - Node.js version:
  - OS:

typedoc: v0.15.0
nodejs: v12.12.0

**Comment by Gerrit0** [comment]
This should be fixed when #1129 is merged.

**Comment by yoboka** [comment]
yep should be fixed but is not published to npm yet..

**Comment by pmconne** [comment]
It is published as a separate package: https://www.npmjs.com/package/@gerrit0/typedoc
I am interested to know when to expect the main typedoc package to be updated though - using this alternative package appears to cause issues loading typedoc plugins.

**Comment by sahilanguralla** [comment]
When is this getting released?

**Comment by Gerrit0** [comment]
@aciccarello and @blakeembrey (and Sebastian Lenz, but he stepped away ages ago) are the only ones with publishing permissions, so I can't promise anything with release dates unfortunately, which is why I set up `@gerrit0/typedoc` to mirror master.

@pmconne any chance you could toss a reproduction up somewhere I could look at? The code there is identical (besides minor changes to change the name + loaded theme), so if it breaks the plugins they are probably broken on master here too, which would be nice to fix.

**Comment by blakeembrey** [comment]
@Gerrit0 I added you as the owner of the NPM package, sorry that I hadn't heard of this issue earlier.

**Comment by Gerrit0** [comment]
Awesome, thank you Blake! I'm working on the release notes now.

**Comment by Gerrit0** [comment]
v0.15.1 is now published with support for 3.7.2 :)
